backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -24,13 +28,12 @@ nav.menu {
 }
 
 .admin-nav {
-  border-right: 1px solid $color-sidebar-border;
   background: $color-sidebar-bg;
   z-index: $z-index-navbar-flyout;
 }
 
 .admin-nav-menu {
-  text-transform: uppercase;
+  text-transform: capitalize;
```

Comment:
Please don't use `capitalize` here, because that changes how every single word looks on the admin tabs. Especially plugin maintainers and users of foreign languages may not want to have every single word capitalized. This should be handled inside the locale files.